lib/MetaCPAN/Model/Archive.pm

```diff
@@ -77,7 +77,11 @@ has _tempdir => (
     init_arg => undef,
     lazy     => 1,
     default  => sub {
-        return File::Temp->newdir;
+
+        my $scratch_disk = '/mnt/scratch_disk';
```

Comment:
I always wonder about hard-coded paths in lexicals like this. Could/should this be in configuration somewhere or else at least a package global?